types/auth0/index.d.ts

```diff
@@ -13,17 +13,36 @@ export interface ManagementClientOptions {
 export interface UserMetadata { }
 export interface AppMetadata { }
 
-export interface UserData {
+export interface CreateUserData {
   connection: string;
   email?: string;
   username?: string;
```

Comment:
It might be a good idea to create a interface these can both extend that has common properties like username, password, email, etc.